tensorbay/cli/utility.py

```diff
@@ -387,3 +395,38 @@ def _set_request_config(config_parser: ConfigParser) -> None:
             client_config.is_internal = config_section.getboolean("is_internal")
         if "max_retries" in config_section:
             client_config.max_retries = config_section.getint("max_retries")
+
+
+def echo_response(err: InternalServerError) -> None:
+    """Log request and response when InternalServerError happens.
+
+    Arguments:
+        err: The InternalServerError raised from the CLI.
+
+    """
+    if logger.disabled:
+        click.echo(dump_request_and_response(err.response))
+
+
+def exception_decorator(cli_func: _Callable) -> Any:
+    """Decorator for CLI functions to catch custom exceptions.
+
+    Arguments:
+        cli_func: The CLI function needs to be decorated.
+
+    Returns:
+        The CLI function with exception catching procedure.
+
+    """
+
+    @wraps(cli_func)
+    def catch_custom_exception(*args: Any, **kwargs: Any) -> None:
```

Comment:
better to name the inner function of a decorator to `wrapper`,
because all other decorators in SDK use that name.